changelog/v0.23.5/remove-panic.yaml

```diff
@@ -0,0 +1,6 @@
+changelog:
+  - type: FIX
+    issueLink: https://github.com/solo-io/gloo/issues/7174
+    resolvesIssue: true
```

Comment:
false perhaps? Since we'll need to pull in these changes to gloo, I assume this won't be true until we have made the change in all repos